changes/add_archiveDetailedQcStatus.md

```diff
@@ -0,0 +1 @@
+Add feature to make Detailed QC Statuses archivable
```

Comment:
Don't need to include the word "Add" since this will be pulled into the [release notes](https://github.com/miso-lims/miso-lims/blob/develop/RELEASE_NOTES.md) under an "Added" heading